delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/Resources.scala

```diff
@@ -314,6 +336,28 @@ object Resources {
 
       }
 
+    def refresh(c: RefreshResource) = {
```

Comment:
It looks really like the one for updated, maybe we could reduce the boilerplate by introducing a common trait that `UpdateResource` and `RefreshResource` would implement ?